lib/src/components/VernaForm/index.spec.tsx

```diff
@@ -150,22 +168,20 @@ describe('VernaForm', () => {
   });
 
   it('should use a selector to query sub schema and add or remove fields on it', async () => {
-    const translations = translationsFactory();
-
     render(
-      <VernaProvider
+      <Wrapper
```

Comment:
Maybe rename the Wrapper so it's obvious that it's only an overload of VernaProvider for the tests? Like just VernaTestWrapper something like this?